Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN export GIT_COMMIT=$(git rev-parse HEAD) && \
             "-X github.com/SAP/jenkins-library/cmd.GitCommit=${GIT_COMMIT} \
             -X github.com/SAP/jenkins-library/pkg/log.LibraryRepository=${GIT_REPOSITORY} \
             -X github.com/SAP/jenkins-library/pkg/telemetry.LibraryRepository=${GIT_REPOSITORY}" \
+        -tags release \
```

Comment:
I would like to suggest to add some docu here with a comment why we have that --tags release. Otherwise somebody after not knowing the details might decide to remove this (e.g. since the build succeeds also without that flag).